hack/README.md

```diff
@@ -149,7 +149,8 @@ pushing directly to a local registry and configuring kind to use this reg, so do
 isnt required.
 - Or a `tilt` recipe which hot reloads all kpng on code changes.
 
-
```

Comment:
for the future, these kind of linters are better integrated in the IDE, we can update the readme on a next moment with the instructions

Reply:
Sure sounds good